airflow/gcp/hooks/gcs.py

```diff
@@ -323,6 +323,26 @@ def delete(self, bucket_name, object_name):
 
         self.log.info('Blob %s deleted.', object_name)
 
+    def delete_bucket(self, bucket_name: str):
+        """
+        Delete an bucket object from the Google Cloud Storage
+
+        :param bucket_name: name of the bucket which will be deleted
+        :type bucket_name: str
+        """
+
+        client = storage.Client()
+        bucket = client.bucket(bucket_name)
+
+        if not bucket.exists():
+            self.log.warning("Bucket {} not exists".format(bucket_name))
```

Comment:
```suggestion
            self.log.warning("Bucket %s not exists", bucket_name)
```
This makes the logs more colorful.

Reply:
Ok, I will do this way. Thanks :)